src/caf.src.js

```diff
@@ -19,6 +19,7 @@
 			this.signal.pr.catch(v=>v);
 		}
 		abort(reason) {
+			this.signal.reason = reason;
```

Comment:
Seems like it might be a good idea to guard against this being reset multiple times... like:

```js
if (!("reason" in this.signal)) {
   this.signal.reason = reason;
}
```